Library/ViewModels/ChangeEmailViewModelSwiftUIIntegrationTest.swift

```diff
@@ -243,6 +244,10 @@ public final class ChangeEmailViewModelSwiftUIIntegrationTest: ChangeEmailViewMo
     self.viewDidLoadProperty.value = ()
   }
 
+  public func didTapSaveButton() {
+    self.saveTriggered.send(true)
```

Comment:
Refactoring this to be a stream of voids would be a slightly bigger project; leaving that for another day.